src/sonic-host-services/scripts/hostcfgd

```diff
@@ -381,6 +382,11 @@ class HostConfigDaemon:
 
 
     def update_all_feature_states(self):
```

Comment:
I feel we should allow self.load() API in the start() to be invoked before the systemctl wait+update_all_feature_states(). The self.load() API does configure the AAA/TACACS.

Reply:
moved the systemctl wait before the update_all_feature_states() in previous commit

Reply:
But `self.load()` still comes after.

Reply:
@judyjoseph Regarding issue fixed by https://github.com/Azure/sonic-buildimage/pull/5740 we explicitly moved load after update_feature.  So if we are waiting more I feel then it's better fir the AAA/TACACS config missing issue.  

Basically all the DB write for AAA/TACACS should happen by the time load() gets callled and now we have add delay so there is better chances for that.

What do you think ?

Reply:
Okay agree that this change should help in (https://github.com/Azure/sonic-buildimage/pull/5740) since we add more delay now. Thanks for bringing it up here.
My point was, if we delay self.load() more, are we indirectly delaying a user getting TACACS server authenticated ?

Reply:
@judyjoseph , sorry I had misunderstood the comment earlier.
This current change will add delay only during bootup, I think it should be ok if the user cannot get authenticated while the system is booting up. 

Let me know if this is ok ?

Reply:
Sure, may be we are ok with that delay as we have to take care of the issue fixed with #5740 ( where the user was not able to login after SONIC conversion as HwProxy TACACS config was not getting programmed)